extension/src/test/XLIFFTypes.test.ts

```diff
@@ -484,38 +485,66 @@ suite("Xliff Types - Functions", function () {
     );
   });
 
-  test("getSameSourceDifferentTarget", function () {
+  test("Xliff.getSameSourceDifferentTarget", function () {
     const xlf = Xliff.fromString(xliffXmlWithDuplicateSources());
     const transUnits = xlf.getSameSourceDifferentTarget(xlf.transunit[1]);
-    assert.equal(
+    assert.deepStrictEqual(
```

Comment:
We've got a ton of changes to do here, I assume... 😁

Reply:
yup :sweat_smile: